rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/90_sparse_vector_mapping.yml

```diff
@@ -0,0 +1,21 @@
+setup:
```

Comment:
I think we should have a simple set of tests for exercising the mapped type & queries and such. No need to have a single file only for mapping tests. 

This file should test:

 - Creating the mapping (in the correct versions)
 - Ensure creation fails in older 8.x versions
 - That queries work
 - That indexing works